edx_name_affirmation/tests/test_handlers.py

```diff
@@ -280,39 +280,48 @@ def test_proctoring_create_verified_name(self):
         verified_name = verified_name_query.first()
         self.assertEqual(verified_name.status, VerifiedNameStatus.PENDING)
 
+    def test_proctoring_does_not_create_name(self):
+        """
+        Test that if we receive a signal for an attempt id that we do not have a name for, we do not create a new
+        record.
+        """
+
+        proctoring_attempt_handler(
+            self.proctoring_attempt_id,
+            self.user.id,
+            'created',
+            None,
+            None,
+            True,
+            True,
+            True
+        )
+
+        self.assertEqual(len(VerifiedName.objects.filter()), 0)
+
     @ddt.data(
-        (None, None, True, True, True),
-        ('John', 'John', False, False, False),
-        ('John', 'John', False, True, True),
-        ('John', 'John', True, True, False)
+        (False, False, False),
```

Comment:
Wanted to separate these test cases out as we actually want to assert that the celery task was not triggered, not just if a verified name was not created.